backend/tournesol/views/stats.py

```diff
@@ -0,0 +1,76 @@
+"""
+API endpoints to show public statistics
+"""
+
+from datetime import timedelta
+
+from django.utils import timezone
+from drf_spectacular.utils import extend_schema, extend_schema_view
+from rest_framework import generics
+from rest_framework.permissions import AllowAny
+from rest_framework.response import Response
+
+from core.models import User
+
+from ..models import Comparison, Video
+from ..serializers import StatisticsSerializer
+
+
+class Statistics:
+    """
+        Representation of a Statistics
+    """
+    def add_user_statistics(self, user_count, last_month_user_count):
+        self.user_count = user_count
+        self.last_month_user_count = last_month_user_count
+
+    def add_video_statistics(self, video_count, last_month_video_count):
+        self.video_count = video_count
+        self.last_month_video_count = last_month_video_count
+
+    def add_comparison_statistics(self, comparison_count, last_month_comparison_count):
+        self.comparison_count = comparison_count
+        self.last_month_comparison_count = last_month_comparison_count
+
+
+@extend_schema_view(
+    get=extend_schema(
+        description="Retrieve statistics"
```

Comment:
We can use a dot at the end.

```suggestion
        description="Retrieve statistics."
```